collector/serverstats.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022 Ben Kochie
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"fmt"
+
+	"github.com/facebook/time/ntp/chrony"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	serverstatsSubsystem = "serverstats"
+)
+
+var (
+	serverstatsNTPHits = typedDesc{
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, serverstatsSubsystem, "ntp_hits"),
+			"Received NTP packets from allowed senders",
```

Comment:
I would prefer to use the long text, verbatim, from the chronyc documentation.

```suggestion
			"The number of valid NTP requests received by the server.",
```

Reply:
Again, I was trying to keep it short but I see the advantages of having a more descriptive description, and consistency with the documentation. I'll change that, probably later today

Reply:
Yea, most of the descriptions are reasonably short. The help doesn't need to be a book, but if it's short enough to copy-n-paste, it's preferable.